cvat/apps/engine/views.py

```diff
@@ -284,11 +282,19 @@ def get_serializer_class(self):
             return ProjectWriteSerializer
 
     def get_queryset(self):
-        queryset = super().get_queryset()
+        queryset = models.Project.objects.select_related('owner', 'assignee', 'organization')
```

Comment:
> otherwise setting class attribute queryset will not make sense

That is not true, this attribute is used to automatically defined ViewSet basename

Reply:
> why did you decide to move select_related('assignee', 'owner') into get_queryset method?

To hold all select/prefetch logic for all actions in one place.

Reply:
How is caching supposed to work here?
Each server request to get project will always do one more request to the database